### DISCUSSION TITLE
oidc config: is it possible to override jwks_uri in dex.config  ?

### DISCUSSION THREAD

**Discussion by sohgaura** [dexidp/dex#3518] [Q&A]
We are planning to use dex for oidc. As we have different issuer and Jwks_uri , is it possible to override jwks_uri retrieved from issuer .well-known/openid-configuration ?  Can you confirm if it will consider jwks_uri passed as parameter or the one retrieved from issuer url? 
Thanks !

**Comment by nabokihms** [comment]
Hello @sohgaura. Should be fixed by https://github.com/dexidp/dex/pull/3527
